guide/how-it-works/ecash/fedimint.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: guide
+title: Fedimint
+description: Learn the basics of Fedimint.
+nav_order: 3
+parent: Ecash
+grand_parent: How it works
+permalink: /guide/how-it-works/ecash/fedimint/
+redirect_from:
+ - /guide/ecash/fedimint
+image: https://bitcoin.design/assets/images/guide/how-it-works/ecash/fedimint.jpg
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Explains generally how fedimint works
+
+Illustration sources
+
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/fedimint.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/fedimint@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/fedimint-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/fedimint-mobile@2x.jpg"
+   alt-text = "ecash overview illustration"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Fedimint
```

Comment:
Sometimes Fedimint is capitalized and sometimes not.

Reply:
ack